itou/approvals/models.py

```diff
@@ -1307,7 +1319,10 @@ def notify_pole_emploi(self, at=None):
                 encrypted_nir,
                 self.siae_siret,
                 type_siae,
-                "PRES",  # hardcoded, PE approvals are assumed as coming from prescribers
+                origine_candidature=job_application_enums.sender_kind_to_pe_origine_candidature(
+                    job_application_enums.SenderKind.PRESCRIBER
+                ),  # hardcoded, PE approvals are assumed as coming from prescribers
+                typologie_prescripteur=prescribers_enums.PrescriberOrganizationKind.PE,
```

Comment:
Peut-être sortir le commentaire car j'ai l'impression qu'il s'applique à `origine_candidature` et `typologie_prescripteur`.
```suggestion
                # hardcoded, PE approvals are assumed as coming from Pôle emploi prescribers
                origine_candidature=job_application_enums.sender_kind_to_pe_origine_candidature(
                    job_application_enums.SenderKind.PRESCRIBER
                ),
                typologie_prescripteur=prescribers_enums.PrescriberOrganizationKind.PE,
```